composition/test/test_composition.py.in

```diff
@@ -0,0 +1,74 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from launch import LaunchDescriptor
+from launch.exit_handler import default_exit_handler
+from launch.launcher import DefaultLauncher
+from launch.output_handler import ConsoleOutput
+from launch_testing import create_handler
+
+
+def test_manual_composition():
+    name = 'test_manual_composition'
+    executable = '@MANUAL_COMPOSITION_EXECUTABLE@'
+    launch(name, [executable])
+
+
+def test_linktime_composition():
+    if os.name == 'nt':
+        print('The link time registration of classes does not work on Windows')
+        return
```

Comment:
Since I don't see an easy way to address this that sounds like a good idea. Done: http://ci.ros2.org/job/ci_windows/1857/testReport/